Source/WebCore/rendering/svg/RenderSVGResource.cpp

```diff
@@ -76,8 +78,7 @@ static inline RenderSVGResource* requestPaintingResource(RenderSVGResourceMode m
 
     bool applyToFill = mode == RenderSVGResourceMode::ApplyToFill;
     SVGPaintType paintType = applyToFill ? svgStyle.fillPaintType() : svgStyle.strokePaintType();
-    if (paintType == SVGPaintType::None)
-        return nullptr;
+    ASSERT(paintType != SVGPaintType::None);
```

Comment:
I think the part of this function till this line can be simplified like this.

```
    bool applyToFill = mode == RenderSVGResourceMode::ApplyToFill;

    if (renderer.view().frameView().paintBehavior().contains(PaintBehavior::RenderingSVGMask)) {
        // When rendering the mask for a RenderSVGResourceClipper, always use the initial fill paint server, and ignore stroke.
        if (!applyToFill)
            return nullptr;

        RenderSVGResourceSolidColor* colorResource = RenderSVGResource::sharedSolidPaintingResource();
        colorResource->setColor(SVGRenderStyle::initialFillPaintColor().absoluteColor());
        return colorResource;
    }

    const auto& svgStyle = style.svgStyle();
    auto paintType = applyToFill ? svgStyle.fillPaintType() : svgStyle.strokePaintType();

    // If we have no fill/stroke, return nullptr.
    if (paintType == SVGPaintType::None)
        return nullptr;
```

Reply:
Done and pushed commit. I think EWS hook will take some time to run it through EWS.